lib/hand.rb

```diff
@@ -13,7 +14,17 @@ def contains_card?(card)
     @cards.include?(card)
   end
 
-  def get_high_card
+  def high_card
     @cards.max_by { |card| card.rank(Deck.FACE_RANKS) }
   end
-end
\ No newline at end of file
+
+  def contains_pair?
+    self.value_hash.has_value?(2)
+  end
+
+  def value_hash
```

Comment:
Cool! I like this helper method. The only suggestion I would have is to give it a better name. It's not just a container for values, it tells us _how many ocurrences_ of each value are in this Hand.